src/compiler/diagnosticMessages.json

```diff
@@ -1759,6 +1759,14 @@
         "category": "Error",
         "code": 2658
     },
+    "'super' in members of object literal expressions is only allowed when option 'target' is 'ES2015' or higher.": {
+        "category": "Error",
+        "code": 2659
+    },
+    "'super' can only be referenced in members of derived classes or object literal expressions.": {

```

Comment:
`"The 'super' keyword can only be referenced in members of derived classes and object literal expressions."`